ta/crypt/include/mbed_taf.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/* Copyright (c) 2018, Linaro Limited */
+
+#ifndef __MBED_TAF_H
+#define __MBED_TAF_H
+
+#include <tee_internal_api.h>
+
+TEE_Result ta_entry_mbed_self_tests(uint32_t param_type,
```

Comment:
I suggest `s/mbed/mbedtls/` and `s/MBED/MBEDTLS/` everywhere (function names, file names, macros) because mbed means nothing. What do you think?

Reply:
Updated